cloudbuild.yaml

```diff
@@ -3,16 +3,12 @@ steps:
   name: gcr.io/$PROJECT_ID/make
   args: [test]
   waitFor: ['-']
-- id: tools/hive-bigquery
-  name: google/cloud-sdk
-  args: [gcloud, builds, submit, tools/hive-bigquery/, --config=tools/hive-bigquery/cloudbuild.yaml]
-  waitFor: ['-']
+# - id: tools/hive-bigquery
```

Comment:
Fine with quarantining this test.
please open a github issue to resolve what's actually going on here and add a comment with link to issue.

Reply:
Done. [issue link](https://github.com/GoogleCloudPlatform/professional-services/issues/532)